src/util/printValue.js

```diff
@@ -1,7 +1,9 @@
 import isFunction from 'lodash/isFunction';
 import isSymbol from 'lodash/isSymbol';
 
-const symbolToString = Symbol.prototype.toString;
+const symbolToString = typeof Symbol !== 'undefined' ?
```

Comment:
Is this for backwards compatibility? Does `typeof` really return `undefined` if the global `Symptom` does not exist? Shouldn't `Symbol` be a function or constructor to have `Symbol.prototype.toString`? I don't understand.

Reply:
I've added the related issue jquense/yup#131 in the commit message.